app/views/vacancies/_vacancy.html.haml

```diff
@@ -1,23 +1,25 @@
-%li.vacancy
+%li.vacancy.bottom-border{ class: 'govuk-!-padding-0'}
   %h2.govuk-heading-m.mb0= link_to vacancy.job_title, job_path(vacancy), class: 'govuk-link view-vacancy-link'
-  %p.govuk-body= vacancy.location
-  %dl
+  %p.govuk-body-s= vacancy.location
+  %dl{ class: 'govuk-!-font-size-14'}
     %dt= t('jobs.salary')
     %dd.double
       = vacancy.salary_range
-    %dt= t('jobs.publish_on')
-    %dd.double
-      = format_date(vacancy.publish_on)
-    %dt= t('jobs.expires_on')
-    - if vacancy.expiry_time.nil?
+    %dt= t('jobs.school_type')
+    - if vacancy.school.has_religious_character?
       %dd.double
-        = format_date(vacancy.expires_on)
-    - else 
+        = vacancy.school.school_type.label + ", " + vacancy.school.gias_data['religious_character']
+    - else
```

Comment:
To save time refactoring this in the future I would create a presenter method that displays the school type and make it reference `vacancy.school.gias_data['TypeOfEstablishment (name)']` and `vacancy.school.gias_data['ReligiousCharacter (name)']`. Then you can remove the if display logic from here as a school will always have a type, so you only need to handle the displaying of the religious character in the presenter method

Reply:
I say this because I think that the school type table will be removed sometime in the future due to the addition of the `gias_data`